saleor/graphql/account/mutations/base.py

```diff
@@ -39,7 +41,21 @@ def can_edit_address(user, address):
     )
 
 
-class SetPassword(CreateToken):
+class AccountErrorMixin:
+    account_errors = graphene.List(
+        graphene.NonNull(AccountError),
+        description="List of errors that occurred executing the mutation.",
+    )
+    ERROR_TYPE_CLASS = AccountError
```

Comment:
Maybe let's just add one empty line above, to visually separate fields declaration from those mixin settings.

Reply:
I've added a line :)